cl/corpus_importer/tests.py

```diff
@@ -716,3 +717,9 @@ def test_short_opinion_matching(self) -> None:
 
         bad_match = compare_documents(harvard_characters, bad_characters)
         self.assertEqual(bad_match, 81)
+
+    def test_florida_court_addition(self):
+        """Can we find a court from our new fixture?"""
+
+        result = courts_db.find_court("Broward County Judges' Court")
+        self.assertEqual(result[0], "flajudct9", msg="Missing Florida Court")
```

Comment:
Will this be a useful test in a week? Is it worth having in our codebase long term? Is there a better test that'd ensure that our code is working properly? It seems like it's just testing that a dependency is upgraded?

Reply:
Removed.